services/pane.test.js

```diff
@@ -1,16 +1,75 @@
 var dirname = __dirname.split('/').pop(),
   filename = __filename.split('/').pop().split('.').shift(),
-  edit = require('./edit'),
   lib = require('./pane'),
   state = require('./page-state'),
+  dom = require('@nymag/dom'),
   ds = require('dollar-slice');
 
+// minimal templates, only what we need to test the logic and functionality
+function stubWrapperTemplate() {
+  return dom.create(`<div class="kiln-toolbar-pane-background">
+    <div class="kiln-toolbar-pane">
+      <span class="pane-header"></span>
+      <div class="pane-inner"></div>
+    </div>
+  </div>`);
+}
+
+function stubMessageTemplate() {
+  return dom.create(`<div class="publish-messages messages">
+    <p class="publish-state-message">In Draft.</p>
+    <p class="publish-schedule-message kiln-hide">Scheduled to publish</p>
+  </div>`);
+}
+
+function stubPublishTemplate() {
+  return dom.create(`<div class="publish-actions actions">
+    <form class="schedule">
+      <label class="schedule-label" for="schedule-date">Date</label>
+      <input id="schedule-date" class="schedule-input" type="date" min="" value="" placeholder=""></input>
+      <label class="schedule-label" for="schedule-time">Time</label>
+      <input id="schedule-time" class="schedule-input" type="time" value="" placeholder=""></input>
+      <button class="schedule-publish">Schedule Publish</button>
+    </form>
+    <button class="unschedule kiln-hide">Unschedule</button>
+    <button class="publish-now">Publish Now</button>
+    <button class="unpublish kiln-hide">Unpublish Page</button>
+  </div>`);
+}
+
+function stubNewPageActionsTemplate() {
+  return dom.create(`<div class="new-page-actions actions">
+    <form class="select-page-type">
+      <button class="create-article-page primary-action">New Article Page</button>
+      <button class="create-sponsored-post primary-action">New Sponsored Post</button>
+    </form>
+  </div>`);
+}
+
+function stubErrorsTemplate() {
+  return dom.create(`<div class="publish-error">
+    <span class="label">There was a problem:</span>
+    <span class="description">Please see below for details</span>
+    <ul class="errors"></ul>
+  </div>`);
+}
```

Comment:
If we do accomplish that dream (mentioned above), maybe we can rethink this level of dom-y testing.